_posts/2018-12-12-dot-css-dot-js-2018.md

```diff
@@ -0,0 +1,292 @@
+
+<img class="image fit" alt="Hex linegraph example" style="max-width: 25%;" src="/img/2018-11-08-dot-css-dot-js-2018/hex-linegraph.jpg">
+<img class="image fit" alt="Line graph of shorthand hex for color #D92" style="max-width: 25%;" src="/img/2018-11-08-dot-css-dot-js-2018/linegraph.jpg">
+
+### Step 3: Hue from shape
+This is what we think  it's the most tricky part about reading color hex codes.
+With the line graph we got from the previous step you'll have to find a matching color on the color wheel.
+So you'll have to remember this one by hearth if you really want to show off ;)
+For our example color it matches best with the color **orange**.
+
+<img class="image fit" alt="HSL color wheel with color names" style="max-width: 75%" src="/img/2018-11-08-dot-css-dot-js-2018/color-wheel.jpg">
+
+### Step 4: Lightness from total
+To determine the lightness you should look at the total sum of the channel values (pairs).
+Or just look at the values in the line graph we created.
+If the values are higher to the top, the color is closer to white and thus lighter.
+If the values are closer to the bottomn, the color is closer to black and thus darker.
+For our example color `#D92`, the values are both high and low, so it has **middle** lightness.
+
+<img class="image fit" alt="Determining lightness of a color" style="max-width: 50%" src="/img/2018-11-08-dot-css-dot-js-2018/lightness.jpg">
+
+### Step 5: Saturation
+Saturation is a measure of how vibrant or rich the hue/color is.
+To measure the saturation whe need to see at the difference between the highest and the lowest value in our shorthand code.
+The wider the range, the higher the saturation. 
+Colors with small range have low saturation, appearing faded.
+A color with no saturation is a pure gray.
+
+<img class="image fit" alt="Measuring and describing saturation of a color" style="max-width: 50%" src="/img/2018-11-08-dot-css-dot-js-2018/saturation.jpg">
+
+
+With our color, #D92, D is the highest value, 2 is the lowest.
+D is high. 2 is low.
+That's a wide range, but not completely wide.
+So our color has moderate saturation, thus making it a **washed** color.
+
+Now we have all three attributes for our color
+So we can say `#D49B25` is **Middle Washed Orange**.
+Pretty amazing right?
+
+<img class="image fit" alt="#D49B25 color described as Middle Washed Orange" style="max-width: 50%" src="/img/2018-11-08-dot-css-dot-js-2018/final-color.jpg">
+
+Since dotCss put's most of it's talks online we definitely recommend watching this talk.
+He just explains it so well and goes a little bit deeper than we did here.  
+
+[View talk](https://www.dotconferences.com/2018/11/david-desandro-read-color-hex-codes){:target="_blank" rel="noopener noreferrer"} | [View slides](https://speakerdeck.com/desandro/read-color-hex-codes){:target="_blank" rel="noopener noreferrer"}
+
+## 'Variable fonts' by [Mandy Michael](https://twitter.com/mandy_kerr){:target="_blank" rel="noopener noreferrer"}
```

Comment:
Maybe give some credit to what she showed us?
She made some pretty nice animations which she showed. 

It think it's pitty we don't say anything about that.